test/index.js

```diff
@@ -1,23 +1,23 @@
 /**
  * Test suite uses mocha and enzyme to mock browser APIs
- * 
- * See Enzyme docs: 
+ *
+ * See Enzyme docs:
  * https://github.com/airbnb/enzyme/blob/master/docs/guides/jsdom.md
  * https://github.com/airbnb/enzyme/blob/master/docs/api/mount.md
  */
 
 const React = require('react');
-const DOM = require('react-dom-factories');
 const sinon = require('sinon');
-const {expect, assert} = require('chai');
-const ReactQuill = require('../src/index');
-const Quill = ReactQuill.Quill;
+const {expect} = require('chai');
+const ReactQuill = require('../lib/index').default; // FIXME
+const Quill = require('../lib/index').Quill; // FIXME
```

Comment:
These could be replaced with `import` by moving to Jest, which supports Babel transpilation out of the box. But it works as-is.

Reply:
Forgot to commit the change. Should be fine now.